src/main/java/org/stellar/sdk/requests/sorobanrpc/SorobanRpcRequest.java

```diff
@@ -0,0 +1,21 @@
+package org.stellar.sdk.requests.sorobanrpc;
+
+import com.google.gson.annotations.SerializedName;
+import lombok.RequiredArgsConstructor;
+import lombok.Value;
+
+@RequiredArgsConstructor
+@Value
+public class SorobanRpcRequest<T> {
```

Comment:
nice, doing the json-rpc protocol manually on top of just an http client here looks good, rather than trying to use an external lib that does the same but would bloat the dependencies, etc.